iis-core/src/test/java/eu/dnetlib/iis/core/IntegrationTestPropertiesReader.java

```diff
@@ -97,4 +127,27 @@ private Properties readUserProperties() {
 		return userProperties;
 	}
 	
+	private String resolveUserPropertiesFilePath() {
+		String userPropertiesPath = System.getProperty("connectionProperties");
```

Comment:
I wonder if the "connectionProperties" name is not too generic. iisIntTestConnectionProperties?